docker-compose-custom.yml

```diff
@@ -6,11 +6,11 @@ services:
     env_file:
       - .env
     ports:
-      - 30336:30333
-      - 9947:9944
-      - 9936:9933
+      - 30333:30333
+      - 9944:9944
+      - 9933:9933
     volumes:
-      - ./scripts:/dhx/scripts
+      - ./scripts:/dhx/node/scripts
```

Comment:
suggest update this to make it specific to the relevant chain.
so if it's for Harbour:

./harbour/scripts:/dhx/node/scripts

or if it's for Local

./local/scripts:/dhx/node/scripts

and change the filename accordingly.
so if its for Harbour, we'd run docker-compose-harbour.yml
or if its for Local, we'd run docker-compose-local.yml

Since docker-compose-custom.yml was initially just meant to be a template for customisation